docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -100,67 +100,67 @@ After attempting to deploy {WebNameTitle} on OpenShift, the Pod is unable to sta
 
 *What it means:*
 
-The resource quota for the OpenShift project has been met, and the Pod is unable to obtain the requested resources.
+The resource quota for the OpenShift project has been met, and the pod is unable to obtain the requested resources.
 
 *How to resolve it:*
 
-Perform either of the following:
+Do either of the following:
 
```

Comment:
Remove "How to resolve it".

"Do either of the following" is not great because it sounds like "Do whatever you want. We don't care." It will have to be reworded when the options are clearer.

The second option, reducing requested resources, makes no sense. If the console does not have sufficient resources, how does reducing the resources resolve the problem? Maybe something is missing here. Check with Jonathan or Marco.